lib/utils.js

```diff
@@ -2,8 +2,16 @@
 
 exports.makeContinuation = (predicate) => {
 
-    return (callback) => {
+    return () => {
```

Comment:
You changed the implementation of `makeContinuation` but didn't change any of the call sites. Also, since this no longer makes a continuation, you should probably either change the name or just remove it and do the promise in-line.

Reply:
I am not sure which call sites you mean. These aren't exported only used internally in _.tasks. Removing the dependency on Items and just using Promise.all was the purpose here.

Also maybe I have the naming wrong but my understanding was it is called a continuation as it continues after the next tick which is still true so not sure what you want me to change this too but happy to take suggestions.

Reply:
A continuation is a callback essentially. This is making something into a promise so you are essentially promisifying it so `Promise.all` works. `makePromise` as a possible name.

Reply:
Ok so your saying we don't need the nextTick anymore or that it should be something like "resolveNextTick"?

Reply:
It can be named `promisify` or something simar.

Reply:
@arb I think you need to look at the code again. If it was just a promisify I would be using the util.promisify in node. 

The point of the original function was to make sure the function didn't return till the "nextTick". Either that is no longer needed or it should be named based on what it does.

Reply:
Yes it does to make it clear to people it isn't just a promisify. I was simply trying to determine if I could just drop the nextTick semantics. Clearly the answer is no so I will name it appropriately. Can you please take a look at my other comments thanks!

Reply:
The `nextTick` is there so that everything inside tasks is async. That's what it's there for. The only current changes required are to rename this function and the https bug mentioned above.